acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/AbstractPage.java

```diff
@@ -0,0 +1,23 @@
+package uk.nhs.digital.ps.test.acceptance.pages;
+
+import org.openqa.selenium.WebDriver;
+import uk.nhs.digital.ps.test.acceptance.webdriver.WebDriverProvider;
+
+/**
+ * Base class for classes implementing 'Page Object' pattern; provides common properties (such as base URL)
+ * and dependencies (such as web driver) to extending classes.
+ */
+abstract class AbstractPage {
+
+    static final String URL = "http://localhost:8080/cms";
```

Comment:
Might be nice to have this as a config parameter to allow it to be easily changed on systems where 8080 is already in use. Is that relatively easy?

Reply:
Indeed, in the future it may be more useful to externalise this URL, although it may also turn out that we never need to do it, so I decided to not to spend time on implementing something we may never need. The default configuration included in the project as generated by Hippo's Maven prototype does the job for now.

Should we need to make this value configurable and, say, have it set to different values depending on the environment, 
it will be reasonably easy to do so - given that the tests' configuration is being managed by Spring we can easily leverage the wealth of the various methods for externalising configuration that Spring supports - properties files, command line arguments, system/environment properties, values passed in from Maven.